test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -55,3 +55,56 @@ Feature: Bind a single application to multiple services
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-2" should be changed to "True"
         And "nodejs-app" deployment must contain SBR name "binding-request-1"
         And "nodejs-app" deployment must contain SBR name "binding-request-2"
+
+        Scenario: Bind two backend services by creating 1 SBR to a single application
+        Given Imported Nodejs application "nodejs-app-2-services-1-sbr" is running
```

Comment:
does this application really uses two databases?

Reply:
NO, the app is not designed that way.